website/docs/docs/on-premises/database.md

```diff
@@ -4,6 +4,10 @@ title: Database
 
 A PostgreSQL database is required for storage of the application data.
 
+:::info
+ If you would like to migrate an existing PostgreSQL database, skip this section and instead follow the guide [**here**](https://docs.pactflow.io/docs/on-premises/installation/migrating).
```

Comment:
It's best to use URLs without the host in them, so they can be tested locally, so just keep from `/docs/` on.

Reply:
Ditto for all the other URLs.